webapp/src/components/OfferView/OfferView.js

```diff
@@ -0,0 +1,349 @@
+  },
+  componentHeader: {
+    padding: 20,
+    margin: 0,
+    width: "100%",
+    display: "flex",
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+  },
+  componentAvatar: {
+    height: "40px",
+    width: "40px",
+  },
+  componentTitleContainer: {
+    width: "100%"
+  },
+  componentTitle: {
+    marginLeft: 17,
+    marginTop: 3,
+    width: "100%",
+    fontFamily: "Roboto",
+    fontsize: "20px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "normal",
+    letterSpacing: "0.15px",
+    textAlign: "left",
+    color: "rgba(0, 0, 0, 0.87)",
+  },
+  componentSubTitle: {
+    marginLeft: 17,
+    marginTop: 1,
+    width: "100%",
+    fontFamily: "Roboto",
+    fontsize: "14px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.43",
+    letterSpacing: "0.25px",
+    textAlign: "left",
+    color: "rgba(0, 0, 0, 0.6)",
+  },
+  componentContent: {
+    overflow: "auto",
+    padding: 20,
+    marginTop: 10,
+    marginBttom: 10,
+  },
+  componentontentText: {
+    fontFamily: "Roboto",
+    fontsize: "14px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.43",
+    letterSpacing: "0.25px",
+    textAlign: "left",
+    color: "rgba(0, 0, 0, 0.6)",
+  },
+  componentontentTextAux: {
+    fontFamily: "Roboto",
+    fontsize: "14px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.43",
+    letterSpacing: "0.25px",
+    textAlign: "left",
+    color: "#121212",
+  },
+  componentActionsButton: {
+    width: "100%",
+    [theme.breakpoints.down('md')]: {
+      position: "absolute",
+      bottom: 0,
+      padding: 10,
+    }
+  },
+  cardActionButton: {
+    position: "absolute",
+    paddingLeft: 16,
+    paddingRight: 16,
+    paddingBottom: 10,
+    paddingTop: 10,
+    bottom: 10,
+    right: 15,
+    borderRadius: "48px",
+    backgroundColor: "#ba0d0d",
+    fontFamily: "Roboto",
+    fontsize: "14px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "normal",
+    letterSpacing: "0.5",
+    textAlign: "center",
+    color: "#ffffff",
+  },
+  iconBottomAppBar: {
+    color: "rgba(0, 0, 0, 0.6);"
+  },
+  stepper: {
+    backgroundColor: "#ffffff",
+  },
+  img: {
+    height: "30vh",
+    objectFit: "cover",
+    overflow: 'hidden',
+    display: 'block',
+    width: '100%',
+  },
+  closeIcon: {
+    position: 'absolute',
+    zIndex: 1,
+    top: 20,
+    right: 20,
+    margin: '0',
+    height: "5vh",
+    '& svg': {
+      fontSize: 25,
+      color: "#121212"
+    }
+  },
+}))
+
+const monthNames = ["January", "February", "March", "April", "May", "June",
+  "July", "August", "September", "October", "November", "December"
+];
+
+const Transition = forwardRef(function Transition(props, ref) {
+  return <Slide direction="up" ref={ref} {...props} />;
+});
```

Comment:
Semicolons here. Please remove them, since they are not necessary in standard js